src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -38,6 +38,7 @@
 using SonarLint.VisualStudio.Core.Analysis;
```

Comment:
As discussed offline, all files that were changed besides ProjectToLanguageMapper and ProjectToLanguageMapperTests contain the same changes which is adding a mock for IConnectedModeSecrets to the construction in testintialize.